app/controllers/admin/payroll_runs_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Admin
+  class PayrollRunsController < BaseAdminController
+    before_action :ensure_service_operator
+
+    def index
+      @payroll_runs = PayrollRun.all.order(created_at: :desc)
+    end
+
+    def new
+      @payroll_run = PayrollRun.new(claims: PayrollRun.payrollable_claims)
+    end
+
+    def create
+      payroll_run = PayrollRun.create!(claim_ids: params[:claim_ids], created_by: admin_session.user_id)
+      redirect_to [:admin, payroll_run]
+    end
+
+    def show
+      @payroll_run = PayrollRun.find(params[:id])
+
+      respond_to do |format|
+        format.html
+        format.csv do
+          csv = Payroll::ClaimsCsv.new(@payroll_run.claims)
+          send_file csv.file, type: "text/csv", filename: "payroll_data_#{@payroll_run.id}.csv"
```

Comment:
I wonder if it would be more useful to us and to Cantium if we include the date of the report in the filename, rather than the ID. That way it's easy to see which month the file corresponds to. The ID is not very useful to anyone other than a developer, and even then we'd need to connect to the database to marry a file against a payroll run record.